config.h.in.cmake

```diff
@@ -151,6 +151,9 @@
 /* Disable cl_khr_fp16 because fp16 is not supported */
 #cmakedefine _CL_DISABLE_HALF
 
+/* Disable cl_khr_fp64 because fp64 is not supported */
```

Comment:
No, this variable is only for host CPU (perhaps i should rename it) and only used in pthread/basic drivers. 

The other drivers each have their own supported extension lists in CMake (used @ building the kernel library). In driver code, CUDA and HSA currently  hardcode double support to fully supported, TCE hardcodes it to `CL_FP_ROUND_TO_NEAREST | CL_FP_INF_NAN`